Failed compiling

In file included from /Users/zhujinli/nc-geo/src/mutable_polygon.cpp:24:
/Users/zhujinli/nc-geo/src/basic_types.h:31:11: fatal error: malloc.h: No such file or directory
   31 | # include <malloc.h>
      |           ^~~~~~~~~~
compilation terminated.

--- a/src/basic_types.h
+++ b/src/basic_types.h
@@ -28,7 +28,6 @@ SOFTWARE.
 #	include <stdlib.h>
 #	include <stdio.h>
 #	include <assert.h>
-#	include <malloc.h>
 #	include <math.h>
 #	include <stddef.h>
 #	include <stdarg.h>

